custom_platforms.js

```diff
@@ -76,6 +76,8 @@ module.exports = {
                 ",hls=",
                 "\\$jsonprune=",
                 ",jsonprune=",
+                "\\$referrerpolicy=",
+                ",referrerpolicy=",
```

Comment:
https://adguard.com/kb/ru/general/ad-filtering/create-own-filters/#referrerpolicy-modifier:~:text=com/abcd.html%5E-,%24referrerpolicy,-%D0%BE%D1%82%D0%BA%D0%BB%D1%8E%D1%87%D0%B0%D0%B5%D1%82%20%D0%B2%D1%81%D0%B5%20%D0%BF%D1%80%D0%B0%D0%B2%D0%B8%D0%BB%D0%B0

Will not be excluded
`@@||example.com/abcd.html^$referrerpolicy`

```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```